examples/evaluation/promptiter/regressionloop/output/sample/optimization_report.md

```
@@ -0,0 +1,112 @@
+|---|---|---|
+| case_drop | pass |  |
+| critical_case | pass |  |
+| estimated_cost | pass |  |
+| evidence_shape | pass |  |
+| hard_failures | pass |  |
+| latency | pass |  |
+| minimum_gain | pass |  |
+| model_calls | pass |  |
+| run_status | pass |  |
+| tokens | pass |  |
+| tool_calls | pass |  |
```

markdownlint-cli2 (0.23.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)